src/main/java/cuchaz/enigma/translation/mapping/serde/EnigmaMappingsReader.java

```diff
@@ -179,8 +179,9 @@ private void readJavadoc(MappingPair<?, RawEntryMapping> parent, String[] tokens
 			throw new IllegalStateException("Javadoc has no parent!");
 		// Empty string to concat
 		String jdLine = tokens.length > 1 ? String.join(" ", Arrays.copyOfRange(tokens,1,tokens.length))  : "";
-		if (parent.getMapping() == null)
-			throw new IllegalStateException("Javadoc requires a mapping!");
+		if (parent.getMapping() == null) {
+			parent.setMapping(new RawEntryMapping(parent.getEntry().getName(), AccessModifier.UNCHANGED));
```

Comment:
maybe add a `todo` on the setter and here so that we in the future update entry mapping to fix this technical debt